.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,5 +25,3 @@ Optional. Ancillary topics, caveats, alternative strategies that didn't work out
 - Start after checking out this branch
 - Include any setup required, such as bundling scripts, restarting services, etc.
 - Include test case, and expected output
-
-Closes #XXX
```

Comment:
I thought this keyword is for closing the linked issue automatically when the PR is merged? Does ZenHub close the issue if a connected PR is merged? Even if it closes it, I still think it might be good to keep this keyword since this looks like a link between an issue and a PR for GitHub itself (and if we are to move to another project mgmt platform again...).

Reply:
Ah yeah that's fair, now that we're using milestones.